### ISSUE TITLE
Non-recoverable error with 1.0.50

### ISSUE THREAD

**Issue by gabfv** [ILLISIS/Constructron-Continued#104]
Hi, I've installed the 1.0.50 update this morning but it causes a non-recoverable error when loading my save:
![image](https://user-images.githubusercontent.com/24780542/209444377-6f7c014a-8f24-4a99-b05c-fbd56a278fff.png)
I've looked at the code where it was specified and it looks like because the job object doesn't have an attempt property? I'm not super familiar with making Factorio mods but I think it needed a migration to apply this property to all currently running jobs?

**Comment by gabfv** [comment]
Oh, is this similar to what is being done for this?
https://github.com/ILLISIS/Constructron-Continued/blob/056c8d2d9c5c064e7b79dffea019a9a8dbb31210/migrations/Constructron-Continued_1.0.27.lua#L1

**Comment by gabfv** [comment]
Oh I was able to fix it with a new migration:
```
-- Fixes for job.attempt nil crash
for job_bundle_index, job_bundle in pairs(global.job_bundles) do
    for index, job in pairs(global.job_bundles[job_bundle_index]) do
        if job.attempt == nil then
            global.job_bundles[job_bundle_index][index].attempt = 1
        end
    end
end
game.print('Constructron-Continued: v1.0.50 migration complete!')
```

**Comment by ILLISIS** [comment]
Thanks! It is fixed in v1.0.51
I almost used this but opted for a smaller change as I was on the fence and in a rush.

**Comment by gabfv** [comment]
Neat, I confirm this is working for 1.0.51!
